crates/configuration/Cargo.toml

```
@@ -6,7 +6,7 @@ edition = "2021"
 [dependencies]
 query-engine-metadata = { path = "../query-engine/metadata" }
 
-ndc-sdk = { git = "https://github.com/hasura/ndc-hub.git", rev = "02d26c1" }
+ndc-sdk = { git = "https://github.com/hasura/ndc-hub.git", rev = "e0e9629" }
```

We can update to RC 16 directly if you like.

```suggestion
ndc-sdk = { git = "https://github.com/hasura/ndc-hub.git", rev = "ee52bae" }
```